tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -484,21 +484,19 @@ object AutoTuner extends Logging {
 
   def loadSystemProperties(fileInput: String): Option[java.util.Map[String, Any]] = {
     val filePath = new Path(fileInput)
-    val fs = Some(FileSystem.get(filePath.toUri, new Configuration()))
-    fs.foreach { dfs =>
-      val yaml = new Yaml()
-      var fsIs: FSDataInputStream = null
-      try {
-        fsIs = dfs.open(filePath)
-        return Some(yaml.load(fsIs).asInstanceOf[java.util.Map[String, Any]])
-      } finally {
-        if (fsIs != null) {
-          fsIs.close()
-        }
+    val fs = FileSystem.get(filePath.toUri, new Configuration())
+    val yaml = new Yaml()
+    var fsIs: FSDataInputStream = null
+    try {
+      fsIs = fs.open(filePath)
```

Comment:
so if this fails this exception goes all the way back to the user?  Does the dataproc code handle it in another way?

Reply:
I tested with passing incorrect file path:
The execution won't fail. The ouput will show the exception in the "comments"

- first test with non-existing file, I get `- System properties file was not found ` in the comments

```
+-------------------------+--------------------+-------------------------------------------------------------------------+
| App Name                | Recommendations    | Comments                                                                |
+=========================+====================+=========================================================================+
| app-20210507103057-0000 | No recommendations | - System properties file was not found                                  |
```


- second test with incorrect FS, I get `- org.apache.hadoop.fs.UnsupportedFileSystemException: No FileSystem for scheme `




```
+-------------------------+--------------------+---------------------------------------------------------------------------------------+
| App Name                | Recommendations    | Comments                                                                              |
+=========================+====================+=======================================================================================+
| app-20210507103057-0000 | No recommendations | - org.apache.hadoop.fs.UnsupportedFileSystemException: No FileSystem for scheme "dgs" |
|                         |                    | - 'spark.executor.memory' should be set to at least 2GB/core.                         |
|                         |                    | - 'spark.executor.instances' should be set to 'num_gpus * num_workers'.               |
|                         |                    | - 'spark.task.resource.gpu.amount' should be set to 1/#cores.                         |
|                         |                    | - 'spark.rapids.sql.concurrentGpuTasks' should be set to 2.                           |
|                         |                    | - 'spark.rapids.memory.pinnedPool.size' should be set to 2g.                          |
+-------------------------+--------------------+---------------------------------------------------------------------------------------+


```